packages/compass-collection/src/components/collection-header/collection-header.tsx

```diff
@@ -23,11 +23,11 @@ import ClusteredBadge from './clustered-badge';
 const collectionHeaderStyles = css({
   paddingTop: spacing[3],
   paddingBottom: spacing[1],
-  minHeight: +spacing[6] + +spacing[1],
+  height: +spacing[6] + +spacing[2],
```

Comment:
To be honest this looks very weird :) Don't we know for sure that spacing value is a number? We could add an extra step to `compass-components` to ensure that it is a number or 0 if someone misused an index.

Reply:
Ah, that's on me, for some reason TS doesn't compile for me locally if I don't explicitly convert these to numbers, if it doesn't cause errors for y'all totally feel free to remove this

Reply:
Ah got ya! You probably have a newer version than we do. From my experience, it is better to keep it then, because later we will all encounter it anyway :)